the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js

```diff
@@ -1,13 +1,38 @@
 import './JoinRoom.css';
+import React, { useState, useEffect } from 'react';
 import { BsFillCaretLeftFill } from "react-icons/bs";
 import { BsFillVolumeUpFill } from "react-icons/bs";
 import { BsFillSendXFill } from "react-icons/bs";
 import { BsFillArrowRightSquareFill } from "react-icons/bs";
-
+import { useLocation } from 'react-router-dom';
 import imgAvatar from '../../assets/avatar-1.svg';
 import imgLogo from '../../assets/gartic-phone.svg';
 import imgNormal from '../../assets/normal.svg';
 const JoinRoom = () => {
+    // useState [id_room, setIdRoom] = (location.state?.id_room);
+    const location = useLocation();
+    // console.log(location.state?.data);
+    const host = location.state?.host;
+    const users = location.state?.data;
+    const id_room = location.state?.id_room;
+    
+
+    
+  
+
+    console.log('host'+host);
+
+    console.log(users);
+    console.log(id_room);
+    
+     
+   
+    const [roomLink, setRoomLink] = useState('');
+    const handleInviteClick = () => {
+        const link =  `http://localhost:3000/${id_room}`;
```

Comment:
Define the URL 'http://localhost:3000' as constant in a config file to reuse and update easily.